src/layouts/TopNav.vue

```diff
@@ -7,6 +7,13 @@
             <h1>lorcana.top</h1>
         </div>
         <nav class="navigation">
+            <div class="menu">
```

Comment:
I updated the nav to dynamically switch to a burger menu using css only. The SVG icon will only show on mobile widths (< 840px).

The menu div can be set to the available space within the parent bar, granting a larger height than the SVG. Using the menu div as the hover target to display the menu allows a buffer zone for the mouse to cross into the popup list without leaving the hover state.
![image](https://github.com/kalebpomeroy/lorcana.top/assets/6663710/16a239ab-3a4f-4d98-9bc2-7f2cb07233ee)